### DISCUSSION TITLE
How to deal with timezones?

### DISCUSSION THREAD

**Discussion by brielov** [prisma/prisma#9297] [Q&A] [answered]
Hi there! I was wondering how to deal with timezones like twitter does for example, and how timezone info can be stored and queried using prisma based on some user's timezone. I've never worry about timezones before so I'm just curious.

**Comment by ryands17** [comment, marked as answer]
@brielov 👋 

Prisma always stores the timezone in UTC so you don't have to worry about the user's timezone. When you want to query for a value based on the user's timezone, the date will be passed from the frontend and your backend should be configured to use UTC so that the date range passed is correct. Prisma will then compare the values correctly based on UTC and then you send the results back to the user.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
